### ISSUE TITLE
[$10][MSFT-227] "FORUM POSTS" link foreground and background colors missed WCAG 2 AA contrast ratio

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2885]
**Steps to Reproduce**
1. Go to: https://www.topcoder.com/members/doremihong
2. Right click > Inspect Elements 
3. Check the "FORUM POSTS" link

**Expected Result**
- Contrast ratio should be 4.5:1
- If you want passes WCAG 2 AA, you can change text color to #757588. New contrast ratio: 4.5:1

**Actual Result**
Foreground Color: #55A5FF
Background Color: #FFFFFF
Contrast Ratio: 2.55:1
=> Therefor, "FORUM POSTS" link foreground and background colors missed WCAG 2 AA contrast ratio

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: AA
**WCAG Success Criterion**: 1.4.3 Contrast (minimum)
**Screen Reader Used**: N/A
**Accessibility Audit Tool used**: Other

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=b9c070b1%2Dbb96%2D40e1%2Db92f%2D16f47db5d911&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fdoremihong%2FTest%20for%20Accessibility%20Challenge%2FBug%5F28

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096241 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096241 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096241 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by PrakashDurlabhji** [comment]
@veshu PR https://github.com/topcoder-platform/community-app/pull/2951

**Comment by PrakashDurlabhji** [comment]
@codeMinter new PR  and new PR raised at topcoder-react-ui-kit

https://github.com/topcoder-platform/community-app/pull/2968

https://github.com/topcoder-platform/topcoder-react-ui-kit/pull/10

**Comment by Dara-K** [comment]
Please use #006dea color for the forum posts

**Comment by PrakashDurlabhji** [comment]
@Dara-K @codeMinter  PR updated according to above.

both PR 
#2968
 https://github.com/topcoder-platform/topcoder-react-ui-kit/pull/10

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096756 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096756 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by lakshmiathreya** [comment]
Verified #006dea for Forum Posts - MacOs Chrome/Edge/Firefox, Win10/Chrome.

**Comment by lakshmiathreya** [comment]
This one looks good for contrast ratio via WAVE and heard via Mac VoiceOver too - pls check on Windows @drasticdpk  @tosha5252

**Comment by tosha5252** [comment]
@lakshmiathreya @drasticdpk looks good on Windows

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30096756<br/><br/>```This is an automated message for crazyk via Topcoder X```
